### PR TITLE
Refactor theming

### PR DESCRIPTION
**Pull request by Pante**
This PR refactors theming.

Changes to scaling textstyles inside widget styles and an extension on BuildContext to provide a `font` getter will be introduce in a subsequent PR.

### CODE REVIEW

File: forui/lib/src/theme/themes.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        secondary: Color(0xFFF4F4F5),
+        secondaryForeground: Color(0xFF18181B),
+        muted: Color(0xFFF4F4F5),
+        mutedForeground: Color(0xFF71717A),
+        destructive: Color(0xFFEF4444),
+        destructiveForeground: Color(0xFFFAFAFA),
+        border: Color(0xFFE4E4E7),
+      ),
+      style: FStyle(
+        borderRadius: BorderRadius.circular(8),
```
Comment (kawaijoe) [inline forui/lib/src/theme/themes.dart:26]: Should we just add a default value for `borderRadius` on `FStyle` so we don't need to keep repeating it.

File: forui/lib/src/theme/themes.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
```
Comment (kawaijoe) [inline forui/lib/src/theme/themes.dart:1]: Just curious, are we avoiding importing `material`?

File: forui_assets/LICENSE [hunk 3]
```diff
@@ -1 +1,40 @@
-TODO: Add your license here.
+--------------------------------------------------------------------------------
+Forui
+
+MIT License
+
+Copyright (c) 2024 Forus Labs
```
Comment (kawaijoe) [inline forui_assets/LICENSE:6]:
```suggestion
Copyright (c) 2024 Forus Labs Pte Ltd
```

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,24 @@
+--------------------------------------------------------------------------------
+Forui
+
+MIT License
+
+Copyright (c) 2024 Forus Labs
```
Comment (kawaijoe) [inline LICENSE:6]:
```suggestion
Copyright (c) 2024 Forus Labs Pte Ltd
```

File: forui/docs/LICENSE
Comment (kawaijoe) [inline forui/docs/LICENSE:1]: I don't think docs should exist inside `forui`. There should be documentation for everything in this repository.

File: forui/LICENSE
Comment (kawaijoe) [inline forui/LICENSE:1]: Might need to update the org name here as well

**Review by kawaijoe — CHANGES_REQUESTED** [review]

File: forui/docs/LICENSE
Comment (Pante) [reply forui/docs/LICENSE:1]: I think I moved it by accident.

**Pull request merged**